instrumental/drivers/motion/_madcitylabs/nanodrive.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+"""
+Created the 03/03/2022
+
+@author: Sebastien Weber
+"""
+from __future__ import division
+
+from instrumental import u, Q_
+from instrumental.log import get_logger
+from instrumental.drivers import Facet
+from instrumental.drivers import ParamSet
+from instrumental.drivers.motion._madcitylabs.nanodrive_midlib import NiceNanodrive, NiceNanoDriveError
+from instrumental.drivers.motion import Motion
+from instrumental.drivers.util import check_units
+
+log = get_logger(__name__)
+
+_INST_PARAMS = ['serial']
+_INST_CLASSES = ['NanoDrive']
```

Comment:
I'm not sure if it's documented very well, but at some point we moved to `_INST_*_` defined as class attributes rather than at the module level. For instance, see the [Tektronix driver](https://github.com/mabuchilab/Instrumental/blob/8cfc5a1f8c93a00a5b23881b170b3fb191bb2087/instrumental/drivers/scopes/tektronix.py).

Reply:
Guess That answer my question above, You should modify the docs concerning this point. Should we still use _INST_CLASSES then?

Reply:
No, you shouldn't need `_INST_CLASSES` anymore. The parser automatically includes classes that define other `_INST_*_` class attributes.

Reply:
got it and modified the doc accordingly